__tests__/app.test.js

```diff
@@ -111,4 +112,52 @@ describe("app", () => {
 
     })
 
+    describe("POST /api/articles/:article_id/comments", () => {
+        test("should return expected user and comment", () => {
+            return request(app).post("/api/articles/2/comments").send({
+                username: 'icellusedkars',
+                body: 'such a big fan wow'
+            }).expect(201).then((res) => {
+
+                const comment = res.body.comment
+
+                expect(comment.body).toBe("such a big fan wow");
+                expect(comment.author).toBe("icellusedkars");
+                expect(comment.article_id).toBe(2);
+                expect(comment.votes).toBe(0);
+
```

Comment:
As we know the exact comment we are getting back here, we can assert against all of the individual properties (you have done this for half of them above). Then we no longer need the match object assertion directly below.